keanacobarde/HHPWsBe
# CLASS CREATION + SEEDING TEST DATA

## Description
- Seeding test data for Orders, Items, and Users
- Creating class structure and folder structure of application
- Completing DBContext and added Program.cs items
- Created structure for API Calls and DTOs

## Related Issue
#2 

## Motivation and Context
- Implemented features are REQUIRED for the completion of this course.

## How Can This Be Tested?
- Pull PR branch to local testing repo
- Ensure all required folders are created, all classes are created, and test data is seeded.
- Ensure DbContext is present with correct seed data
- Ensure all needed entries are made to Program.cs

## Screenshots (if appropriate):
N / A

## Types of changes
